q2mm/gradient.py

```diff
@@ -243,13 +245,7 @@ def run(self, ref_data=None, restart=None):
             #                  (ref_data[i].val - self.ff.data[i].val)
             count = 0
             for data_type in data_types:
-
-              
-                if (sys.version_info > (3,0)):
-                    rc_zip = zip(r_dict[typ],c_dict[typ])
-                else:
-                    rc_zip = itertools.izip(r_dict[data_type],c_dict[data_type])
-                for r,c in rc_zip:
+                for r,c in zip(r_dict[typ],c_dict[typ]):
```

Comment:
I'd prefer `for r, c in zip(r_dict[typ, c_dict[typ]):`, but it's so minor.

Reply:
You mean the space between `r_dict[typ]` and `c_dict[typ]`? I actually prefer that style too, but this just slipped by me. Sometime in the future I may go through and do a cleanup pull request when I tackle a lot of things like that. I've also noticed unused imports so that is another thing that needs cleaned up.